dataland-frontend/tests/component/testUtils/ExistenceChecks.ts

```diff
@@ -14,7 +16,41 @@ export function checkImage(name: string, fileName: string): void {
  * Checks if a button is present
  * @param name the "name" identifier of the button
  * @param message the string expected to be contained
+ * @param parentElement the element whose content to search for the button
+ * @returns the chainable on the button element
  */
-export function checkButton(name: string, message: string): void {
-  cy.get(`button[name=${name}]`).should("be.visible").should("contain.text", message);
+export function checkButton(
+  name: string,
+  message: string,
```

Comment:
Maybe better "text" instead of message. Or "buttonText".               "message" confused me a little bit at first glance

Reply:
Done